toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -166,24 +168,27 @@ func resolveSingleNode(cloner *rpmrepocloner.RpmRepoCloner, node *pkggraph.PkgNo
 		return
 	}
 
-	desiredPackage := &pkgjson.PackageVer{
-		Name: resolvedPackage,
-	}
+	if !fetchedPackages[resolvedPackage] {
+		desiredPackage := &pkgjson.PackageVer{
+			Name: resolvedPackage,
+		}
 
-	err = cloner.Clone(cloneDeps, desiredPackage)
-	if err != nil {
-		logger.Log.Errorf("Failed to clone '%s' from RPM repo. Error: %s", resolvedPackage, err)
-		return
+		err = cloner.Clone(cloneDeps, desiredPackage)
+		if err != nil {
+			logger.Log.Errorf("Failed to clone '%s' from RPM repo. Error: %s", resolvedPackage, err)
+			return
+		}
+		fetchedPackages[resolvedPackage] = true
+
+		logger.Log.Infof("Fetched: %s", resolvedPackage)
```

Comment:
Nit: I know it's already been this way, but won't logging this on an info level be a bit noisy and not super helpful for regular use anyway? What do you think about moving it to debug level?

Reply:
This was intentionally set at INFO level recently: https://github.com/microsoft/CBL-Mariner/pull/401 (see 3 in the summary)